bin/create_oe_depedencies.sh

```diff
@@ -15,6 +15,7 @@ tar --create --verbose --gzip --absolute-names --show-transformed-names --derefe
 	"$IMX6_ROOTFS/usr/lib/arm-linux-gnueabihf/libQtGui.so.4" \
 	"$IMX6_ROOTFS/usr/lib/arm-linux-gnueabihf/libQtNetwork.so.4" \
 	"$IMX6_ROOTFS/usr/lib/arm-linux-gnueabihf/libSM.so.6" \
+	"$IMX6_ROOTFS/usr/lib/arm-linux-gnueabihf/libcec.so" \
```

Comment:
I think this file is superfluous. The QT dependencies are still QT4

Reply:
I will remove in next cleanup then

Reply:
Thank you.